src/languages/regex.js

```diff
@@ -65,30 +65,14 @@ export default function (hljs) {
     begin: `\\(\\?(?:${MODIFIERS})\\)`
   };
 
-  let GROUP = {};
-  GROUP = {
-    className: 'group',
-    begin: `\\((?:\\?(?:${GROUP_SPECIALS.join('|')}))?`, end: /\)/,
-    // starts: {
-    //   // begin: /./, end: /\)/,
-    //   // excludeBegin: true,
-    //   contains: [
-    //     ESCAPE_SEQUENCE,
-    //     CHARACTER_CLASS,
-    //     META,
-    //     COMMENT,
-    //     MODIFIER,
-    //     GROUP
-    //   ]
-    // }
-    contains: [
-      ESCAPE_SEQUENCE,
-      CHARACTER_CLASS,
-      META,
-      COMMENT,
-      MODIFIER,
-      'self'
-    ]
```

Comment:
Actually we probably don’t need to match nesting groups. Instead, we just need to match the beginning and end of groups. If the group delimiters don’t match, so what?